docs/http_api.rst

```diff
@@ -95,3 +95,30 @@ API Methods
 
    :statuscode 200: opt out deleted
    :statuscode 404: there's nothing to delete
+
+.. http:get:: /optouts/count
+
+   Return the number of opt out
```

Comment:
I think this should say `Return the number of opt outs.` -- note the fullstop and plural of opt outs.